pyproject.toml

```diff
@@ -8,15 +8,19 @@ authors = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-    "sigtools>=4.0.1",
     "typing-extensions>=4.12.2",
 ]
+
 classifiers = [
   "Operating System :: OS Independent",
   "License :: OSI Approved :: Apache Software License",
   "Programming Language :: Python :: 3",
 ]
 
+[project.optional-dependencies]
+compile = [
```

Comment:
Would [dev] be more obvious?

Reply:
I considered it, but there is also the `dev` "dependency-group" (different concept than extras, weirdly enough) which encodes which dependencies a *developer of synchronicity* needs, and that group name seems pretty standard (`uv add --dev` etc.)

Reply:
ok that's fair, i wasn't thinking clearly about the distinction between "developing a library that is using synchronicity" and "developing synchronicity"

(although in practice of course that venn diagram is a circle 😁)

Reply:
> ok that's fair, i wasn't thinking clearly about the distinction between "developing a library that is using synchronicity" and "developing synchronicity"
> 
> (although in practice of course that venn diagram is a circle 😁)

😄